sycl/doc/UsersManual.md

```diff
@@ -290,6 +290,33 @@ and not recommended to use in production environment.
 
     NOTE: This flag is currently only supported with the CUDA and HIP targets.
 
+
+**`-f[no-]sycl-esimd-force-stateless-mem`** [EXPERIMENTAL]
+
+    Enforces usage of stateless memory accesses and automatic conversion of
+    "stateful" memory accesses via SYCL accessors to "stateless" accesses
+    within ESIMD (Explicit SIMD) kernels by compiler.
```

Comment:
```suggestion
    Enforce stateless memory access and automatic conversion of "stateful" 
    memory access via SYCL accessors to "stateless" accesses within ESIMD
    (Explicit SIMD) kernels.
```

Reply:
I updated this section here: fc687f617ef5a730bcd9436e4d6cdd5ea33354ef